frontend/src/views/Room.jsx

```diff
@@ -315,6 +315,12 @@ function Room() {
     setIsSharingScreen(!isSharingScreen);
   };
 
+  useEffect(() => () => {
```

Comment:
does it work without the second `() =>`?

Reply:
hey Pato! nice catch! I was googling and I found out that this is equivalent to write this:
`  useEffect(() => { return () => { if (isSharingScreen) { updateScreenShare(); } }; }, [isSharingScreen]);`

Turns out when you make a double arrow function on a useEffect that function only will be triggered like a cleanup function.
```suggestion
  useEffect(() => () => {
```